### PR TITLE
MCKIN-12927 | make the Cross Icon Visible on IE x block popup.

### PR DESCRIPTION
**Pull request by hussnain2**
MCKIN-12927 | make the Cross Icon Visible and arrange the sequence for accessibility.

connects https://github.com/edx-solutions/xblock-image-explorer/pull/74

### CODE REVIEW

File: lms/static/sass/xblocks/image_explorer.scss [hunk 1]
```diff
@@ -177,10 +177,12 @@ body.new-theme {
               &:hover {
                 background-color: transparent !important;
               }
-              i{
+              &:after {
                 font-family: "Material Icons";
+                content: "\e14c";
```
Comment (nasirhjafri) [inline lms/static/sass/xblocks/image_explorer.scss]: I think we were removing content attributes as part of Accessibility.

Comment (hussnain2) [reply lms/static/sass/xblocks/image_explorer.scss]: @nasirhjafri you are Right, but it's not a new style that we are including, it just reverts back the code. So it, not a primary requirement of this ticket, where we are using .sr class for the Screen Reader.

Comment (nasirhjafri) [reply lms/static/sass/xblocks/image_explorer.scss]: Yes, please use the HTML element instead.

Comment (nasirhjafri) [reply lms/static/sass/xblocks/image_explorer.scss]: I'll let someone else review, @msaqib52  can you please review it? Thanks

Comment (msaqib52) [reply lms/static/sass/xblocks/image_explorer.scss]: @hussnain2 if there are design differences between stock and mckinsey IE feedback close button than it is ok to add it as after element, As long as the button is focusable by keyboard and screen readers read this button.

Comment (hussnain2) [reply lms/static/sass/xblocks/image_explorer.scss]: @msaqib52  yes you got the point, Actually, there is a Design difference between stock and Mckinsey IE feedback close button. meanwhile, it's still focusable.

Comment (msaqib52) [reply lms/static/sass/xblocks/image_explorer.scss]: ok thanks in that case it is ok, Approving the PR

**Pull request merged**